src/hotspot/share/classfile/vmSymbols.hpp

```diff
@@ -347,6 +347,7 @@
   /* common method and field names */                                                             \
   template(object_initializer_name,                   "<init>")                                   \
   template(class_initializer_name,                    "<clinit>")                                 \
+  template(inline_factory_name,                       "<new>")                                    \
```

Comment:
In JDK-8247569 there's an uncertainty about the static factory name, but the CR mentions "\<vnew\>", not "\<new\>" (the change would need to be propagated to all comments).